instrumentation/rocketmq/rocketmq-client/rocketmq-client-4.8/library/README.md

```diff
@@ -0,0 +1,47 @@
+# Manual Instrumentation for Apache RocketMQ Remoting-based Client version 4.0.0+
+
+Provides OpenTelemetry instrumentation for [Apache RocketMQ](https://rocketmq.apache.org/) Remoting-based Client.
+
+## Quickstart
+
+### Add these dependencies to your project:
+
+Replace `OPENTELEMETRY_VERSION` with the latest stable
+[release](https://mvnrepository.com/artifact/io.opentelemetry). `Minimum version: 1.1.0`
+
+For Maven, add to your `pom.xml` dependencies:
+
+```xml
+
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-rocketmq-client-4.8</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+</dependencies>
+```
+
+For Gradle, add to your dependencies:
+
+```groovy
+implementation("io.opentelemetry.instrumentation:opentelemetry-rocketmq-client-4.8:OPENTELEMETRY_VERSION")
+```
+
+### Usage
+
+Remoting-based Client of RocketMQ provides the native interceptor to register the message hook in the instrumentation library.
+
```

Comment:
What does the example below does? Could you run us through it?

Reply:
OK

Reply:
I think it can be very simple, see https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/okhttp/okhttp-3.0/library/README.md#usage for example